src/ray/common/common_protocol.h

```diff
@@ -104,15 +104,13 @@ string_vec_to_flatbuf(flatbuffers::FlatBufferBuilder &fbb,
 template <typename ID>
 flatbuffers::Offset<flatbuffers::String> to_flatbuf(flatbuffers::FlatBufferBuilder &fbb,
                                                     ID id) {
-  return fbb.CreateString(reinterpret_cast<const char *>(id.data()), sizeof(ID));
+  return fbb.CreateString(reinterpret_cast<const char *>(id.data()), kUniqueIDSize);
```

Comment:
I'd like to add `kUniqueIDSize` as a static member of `ID` class, therefore, we can use it like the following code which looks more nature.
```C++
return fbb.CreateString(reinterpret_cast<const char *>(id.data()), id.kUniqueIDSize);
``` 
or
```C++
return fbb.CreateString(reinterpret_cast<const char *>(id.data()), ID::kUniqueIDSize);
```

Reply:
we can use id.size() in this place.